src/lib.rs

```diff
@@ -6,7 +6,7 @@
  **********************************************************************************************************************/
 #![doc(html_root_url = "https://docs.rs/ruspiro-singleton/0.3.0")]
 #![no_std]
-#![feature(asm)]
+#![feature(llvm_asm)]
```

Comment:
Hey, it seems there is no inline assembly used in this crate at all. I've overlooked this in the past. So we can completely remove the usage of this feature.